src/Microsoft.Health.Fhir.Shared.Client/FhirClient.cs

```diff
@@ -283,6 +283,19 @@ public async Task<Uri> ExportAsync(string path = "", string parameters = "", Can
             return response.Content.Headers.ContentLocation;
         }
 
+        public async Task<Uri> AnonymizedExportAsync(string anonymizationConfig, string etag = null, CancellationToken cancellationToken = default)
+        {
+            using var message = new HttpRequestMessage(HttpMethod.Get, $"$export?_anonymizationConfig={anonymizationConfig}&_anonymizationConfigEtag={etag}");
```

Comment:
Do we need to URL encode the query parameter values?

Reply:
Thanks, add encode to be safe here.